portfolio/src/main/webapp/script.js

```diff
@@ -44,3 +44,58 @@ function addDataServletToDom(servlet) {
     quoteContainer.innerText = servlet;
 }
 
+function getComments(){
+    console.log("Fetching comments");
+    const responsePromise = fetch('/data');
+    // console.log(responsePromise);
+    const jsonPromise = responsePromise.then(handleResponse1);
+    console.log(jsonPromise);
+    jsonPromise.then(handleComments);
+}
+
+
+function handleResponse1(response){
+    console.log(response);
+    return response.json();
+    // console.log("Handle Response");
+    // const textPromise = response.text();
+    // textPromise.then(addCommentsToDom)
+}
+    
+// // fetch the /server-stats URL
+// const fetchPromise = fetch('/server-stats');
+
+// // after the fetch comes back, call the handleResponse function,
+// // which itself returns a Promise
+// const jsonPromise = fetchPromise.then(handleResponse);
+
+// // when the jsonPromise comes back, call handleStats with the result
+// jsonPromise.then(handleStats);
+
+// function handleResponse(response) {
+//   return response.json();
+// }
+
+function handleComments(comments) {
+// your code
+    console.log(comments);
```

Comment:
Delete this log statement before submitting. In general, the only log statements you want to leave in are those that would be helpful for someone else to see when running your code